frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -563,6 +567,28 @@ run_test('initialize', () => {
         actual_value = options.updater(othello, click_event);
         assert.equal(appended_name, 'Othello, the Moor of Venice');
 
+        var appended_names = [];
+        people.get_person_from_user_id = function (user_id) {
+            var users = {100: hamlet, 104: lear};
+            return users[user_id];
+        };
+        people.my_current_email = function () {
+            return 'hamlet@zulip.com';
+        };
+        compose_pm_pill.set_from_typeahead = function (item) {
+            appended_names.push(item.full_name);
+        };
+
+        var cleared = false;
```

Comment:
@synicalsyntax

The CI fails due to the `cleared` variable never been used.
If the `cleared` value is really not to be used, replace `clear_text` below with `function(){}`

Reply:
Thanks for catching this (don't know why I forgot to commit that line), tests are now passing